[BUGFIX] Le polyfill randomUUID est absent du packaging (PIX-10976)

aurelie-crouillebois opened the pull request:
## :unicorn: Problème

La fonction randomUUID est utilisée par l'une de nos librairies (ember-data/store), mais est absente des anciens navigateurs (voir [ici](https://caniuse.com/?search=randomUUID)).
L'ajout d'un polyfill est préconisé par ember-data pour palier à cette absence.
La configuration permettant cet ajout de polyfill n'est plus fonctionnelle depuis la version 4.12.0 d'ember-data. 

## :robot: Proposition
Après [investigation](https://github.com/emberjs/data/issues/8675), l'ajout explicite de la librairie @embroider/macros est nécessaire pour ajouter le polyfill.

## :rainbow: Remarques
La version de la librairie @embroider/macros était surchargée dans notre configuration pour une incompatibilité avec une librairie mirage. Cette surcharge a été supprimée car la librairie posant problème avait été déjà mise à jour.

## :100: Pour tester
Sur lambdatest, ouvrir l'application 1d avec une version de navigateur ne supportant pas randomUUID (https://caniuse.com/?search=randomUUID). Le bouton "Je vérifie" doit être fonctionnel (le message de réussite ou d'échec est affiché).

pix-bot-github commented:
Une fois les applications déployées, elles seront accessibles via les liens suivants :
  * [App (.fr)](https://app-pr8120.review.pix.fr)
  * [App (.org)](https://app-pr8120.review.pix.org)
  * [Orga (.fr)](https://orga-pr8120.review.pix.fr)
  * [Orga (.org)](https://orga-pr8120.review.pix.org)
  * [Certif (.fr)](https://certif-pr8120.review.pix.fr)
  * [Certif (.org)](https://certif-pr8120.review.pix.org)
  * [1D](https://1d-pr8120.review.pix.fr)
  * [Admin](https://admin-pr8120.review.pix.fr)
  * [API](https://api-pr8120.review.pix.fr/api/)
  * [Audit Logger](https://pix-audit-logger-review-pr8120.osc-fr1.scalingo.io/api/)

Les variables d'environnement seront accessibles via les liens suivants :
  * [scalingo front](https://dashboard.scalingo.com/apps/osc-fr1/pix-front-review-pr8120/environment)
  * [scalingo api](https://dashboard.scalingo.com/apps/osc-fr1/pix-api-review-pr8120/environment)
  * [scalingo audit-logger](https://dashboard.scalingo.com/apps/osc-fr1/pix-audit-logger-review-pr8120/environment)

aurelie-crouillebois committed:
fix(1d): add @embroider/macros to direct dependencies

@embroider/macros needs to be directly added to project dependencies to make polyfills
to be added into the packaging.

This makes randomUUID function to be available for ember-data store.

yaf reviewed (approved):
:rocket:
